ui/components/multichain/notification-list-item/notification-list-item.tsx

```diff
@@ -143,18 +182,27 @@ export const NotificationListItem = ({
             {formatMenuItemDate(createdAt)}
           </Text>
           {/* Amount */}
-          {amount && (
+          {'amount' in restProps && (
```

Comment:
I guess it would've also worked if amount and cta were destructured earlier so you wouldn't need to update these lines

```
const { amount, cta } = restProps
```

Reply:
Yeah I refactored the types above. Making descructuring not as easy.

```
type BaseNotification = BaseProps;
type NotificationToken = BaseProps & { amount: string };
type NotificationPlatform = BaseProps & {
  cta?: { content: string; link: string };
};
export type NotificationListItemProps =
  | BaseNotification
  | NotificationToken
  | NotificationPlatform;
```

I'm planning on having a stricter discriminator value to help me determine different items.